index.html

```diff
@@ -8,12 +8,145 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <script src="https://kit.fontawesome.com/a6d80728cd.js" crossorigin="anonymous"></script>
 </head>
 <body>
+      
+
+
+    <header>
+                <img class="header_logo" id="logo"
+        src="img/karma-logo.svg"
+        alt="Karma logo"
+          />
+      <nav>
+        <a href="">Meet Karma</a>
+        <a href="">How it Works</a>
+        <a href="/store.html">Store</a>
+        <a href="">Blog</a>
+        <a href="">Help</a>
+        <a href="">Login</a>
+      </nav>
+    </header>
+
+    <main>
+
+        <section>
+      <section class="first_section">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go</p>
+        <button>Learn More</button>
+      </section>
+
+      <section class="second_section">
+
+        <div1>
+        <h2>Everyone needs a little Karma</h2>
+        </div>
+        
+          <div class="icon_banner">  
+            <a href="#">
+            <img
+            src="img/icon-devices.svg"
+            alt="icon-devices"
+            width="150"
+            height="150"
+            class="icons"
+            />
+            <a>Internet for all devices</a>
+          </div>
+
+          <div class="icon_banner">  
+            <a href="#">
+            <img
+            src="img/icon-coffee.svg"
+            alt="icon-devices"
+             width="150"
+            height="150"
+            class="icons"
+            /> <a>Boost your productivity
+            </a>
+          </div>
+
+          <div class="icon_banner"> 
+            <a href="#">
+            <img
+            src="img/icon-refill.svg"
+            alt="icon-devices"
+             width="150"
+            height="150"
+            class="icons"
+            /> <a>Pay as you go
+            </a>
+          </div>
+
+      </section>
+      <div class="extended_feature">
+        <div class="homepage_feature">
+          <img src="/img/homepage-feature.png" alt="homepage-feature" />
+        </div>
+        <div class="text-button-container">
+          <h2>
+            <span style="color:#E0633A;">"</span>Wherever I am, I just dont worry about my connection anymore!<span style="color:#E0633A;">"</span></h2>
+          <button>Get Karma today</button>
+        </div>
+      </div>
+
+           
+            
+        </main>
+
+
+        <footer class="footer">
```

Comment:
using `class` attribute here is redundant because can directly target the `footer` element in your CSS.